JSDemos/Demos/PivotGrid/SummaryDisplayModes/description.md

```diff
@@ -1 +1,19 @@
-The PivotGrid component supports several modes for summary calculation, such as&nbsp;a&nbsp;variation or&nbsp;percent of&nbsp;a&nbsp;total value. Select the required mode from the context menu appearing when you right-click the &laquo;Relative Sales&raquo; field.
\ No newline at end of file
+Summaries are values displayed in pivot grid cells. You can configure summaries in the [fields](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/) that belong to the *"data"* area (`Relative Sales` in this demo). 
+
+## Available Summary Display Modes
+You can specify a post-processing function to modify summaries after they are calculated. Use the [summaryDisplayMode](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#summaryDisplayMode) property to specify how to display summaries. This property allows you select one of the predefined summary display modes. To implement a custom mode, use the [calculateSummaryValue](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#calculateSummaryValue) property. Note that you cannot use predefined and custom modes simultaneously. 
+
+The list below shows available predefined modes. These modes calculate summaries depending on previous row values or totals:
+
+- *"absoluteVariation"*
+- *"percentVariation"*
+- *"percentOfColumnTotal"*
+- *"percentOfRowTotal"*
+- *"percentOfColumnGrandTotal"*
+- *"percentOfRowGrandTotal"*
+- *"percentOfGrandTotal"*
+
+Refer to the [description](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#summaryDisplayMode) of the **summaryDisplayMode** property to find out more about each mode.
+
+## Change the Summary Display Mode at Runtime
+This demo shows how to switch between the above mentioned modes. Right-click or long-tap the 'Relative Sales' button in the [field panel](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/FieldPanel) to invoke the context menu. In this demo, the [onContextMenuPreparing](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#onContextMenuPreparing) handler is used to specify the context menu content. This handler modifies fields from the **PivotGridDataSource**.**fields[]** array and changes their display [format](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#format). 
```

Comment:
```suggestion
To allow users to switch between predefined modes, specify the [onContextMenuPreparing](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#onContextMenuPreparing) handler. Use it to add the corresponding menu items for the field whose area property is set to _"data"_. Make sure that the [field panel](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/FieldPanel) is [visible](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/fieldPanel/).
```

The format prop. is secondary.

Reply:
>To switch between predefined modes, users can right-click a field whose [area](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#area) is set to _"data"_ .

Seems like it is not technically correct. Have you tried this?

>Use the [onContextMenuPreparing](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#onContextMenuPreparing) handler to add specific modes for such a field.

We can use any component (even none-DX ones) to display the list of these modes. So the last sentence - 'n this demo, you can right-click the 'Relative Sales' field to test this feature.' - relates to everything about **onContextMenuPreparing**